0009-simple-payment-setup-protocol/0009-simple-payment-setup-protocol.md

```diff
@@ -191,106 +142,23 @@ Content-Type: application/json
-}
-```
-
-| Field | Type | Description |
-|---|---|---|
-| `type` | `"invoice"` | Receiver type |
-| `account` | ILP Address | ILP Address of the recipient's account |
-| `currency_code` | String | Currency code to identify the receiver's currency. Currencies that have [ISO 4217](https://en.wikipedia.org/wiki/ISO_4217) codes should use those. Sender UIs SHOULD be able to render non-standard codes |
-| `currency_symbol` | String | Symbol for the receiver's currency intended for display in the sender's UI (e.g. `"$"` or `"shares"`). Sender UIs SHOULD be able to render non-standard symbols |
-| `amount` | Decimal String | Value of the invoice in the recipient's currency |
-| `status` | Enum: `"paid"`, `"unpaid"`, `"cancelled"` | State of the invoice |
-| `invoice_info` | URI | URI where additional information about the invoice can be found. |
-
-### Setup (`POST <receiver>`)
-
-When the sender is ready to make a payment, it submits a payment object to the receiver.
-
-#### Request
-
-##### For a Payee
-
-``` http
-POST /api/receiver/bob HTTP/1.1
-Host: red.ilpdemo.org
-Accept: application/json
-
-{
-  "amount": "10.40",
-  "sender_identifier": "alice@blue.ilpdemo.org",
-  "memo": "Hey Bob!"
-}
-```
-
-| Field | Type | Description |
-|---|---|---|
-| `amount` | Decimal String | _Destination amount_ the receiver will receive |
-| `sender_identifier` | String | Identifier of the sender |
-| `memo` | String | Message for the recipient linked to the payment |
+### Payment Setup
 
-##### For an Invoice
+The sender uses the receiver details to create the ILP packet:
+* The `account` in the ILP packet is the `destination_account` provided by the receiver
+* The `amount` is determined by the sender:
 
-``` http
-POST /api/receiver/bob HTTP/1.1
-Host: red.ilpdemo.org
-Accept: application/json
+  * The sender uses the `ledger_info.scale` to determine the integer amount to include in the ILP packet (e.g. if they want the recipient to receive 10 units on their ledger, a `ledger_info.scale` value of 2 would indicate they should set the `amount` to 1000). In cases where the sender knows the integer amount the receiver should get, for example if that amount was requested by the receiver out of band, the sender does not need to use the `ledger_info.scale`
+  * The sender SHOULD NOT set an amount greater than the `maximum_destination_amount` or lower than the `minimum_destination_amount`, as this will be rejected by the receiver
+  * The sender SHOULD NOT set an amount with a precision greater than the `ledger_info.precision`, as this will be rejected by the destination ledger
 
-{
-  "sender_identifier": "alice@blue.ilpdemo.org"
-}
-```
+* The `data` is encoded using the [Pre-Shared Key protocol](../0016-pre-shared-key/0016-pre-shared-key.md):
 
-| Field | Type | Description |
-|---|---|---|
-| `sender_identifier` | String | Identifier of the sender |
+  * The key is the `shared_secret` provided by the receiver
```

Comment:
As above, nested lists must be indented 4 spaces more than the parent list for most Markdown parsers to recognize them as such.